jdi-light/src/main/java/com/epam/jdi/light/driver/get/DriverData.java

```diff
@@ -222,15 +236,14 @@ public static void setupCapability(ChromeOptions cap, String property, String va
             () -> cap.setCapability(ACCEPT_SSL_CERTS, true));
         // Capabilities from settings
         CAPABILITIES_FOR_IE.forEach(cap::setCapability);
-    };
-    public static JAction1<EdgeOptions> EDGE_OPTIONS = cap -> {
+    }
+    public static JAction1<InternetExplorerOptions> IE_OPTIONS = DriverData::defaultIEOptions;
 
-    };
-    public static JAction1<OperaOptions> OPERA_OPTIONS = cap -> {
+    public static void defaultEdgeOptions(EdgeOptions cap) { }
+    public static JAction1<EdgeOptions> EDGE_OPTIONS = DriverData::defaultEdgeOptions;
 
-    };
-    // GET DRIVER
-    public static JFunc1<WebDriver, WebDriver> DRIVER_SETTINGS = DriverData::maximizeScreen;
+    public static void defaultOperaOptions(OperaOptions cap) { }
+    public static JAction1<OperaOptions> OPERA_OPTIONS = DriverData::defaultOperaOptions;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Fields should be declared at the top of the class, before any method declarations, constructors, initializers or inner classes.](https://app.codacy.com/manual/42_/jdi-light/pullRequest?prid=4958360)